app/assets/stylesheets/application.css.scss

```diff
@@ -3,5 +3,11 @@
 */
 
 @import "libraries-main";
+@import "fonts";
+@import "partials/_variables";
+@import "partials/_alerts";
 @import "partials/_facets";
+@import "partials/_panels";
 @import "partials/_search";
+@import "partials/_typography";
+@import "timdexui";
```

Comment:
I checked our other rails apps out of curiosity, and I noticed that in Bento the 'main' stylesheet is also a partial, whereas in Thing there are no partials. Not recommending a change here, since this approach makes logical sense to me, but I wonder if that reaches the level of needing a team discussion about consistency in how we load stylesheets.